src/modules/Providers/views/details/tabs/VirtualMachines/utils/helpers/getCategoryTitle.tsx

```diff
@@ -11,7 +13,7 @@ import {
  * @param {TFunction} t - The translation function.
  * @returns {string} The title for the given category.
  */
-export const getCategoryTitle = (category: string, t): string => {
+export const getCategoryTitle = (category: string, t: TFunction): string => {
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from args